src/ui/public/share/components/share_context_menu.tsx

```diff
@@ -132,25 +133,25 @@ export class ShareContextMenu extends Component<Props> {
         id: panels.length + 1,
         title: `Share this ${this.props.objectType}`,
         items: menuItems
-          .map(menuItem => {
-            menuItem['data-test-subj'] = `sharePanel-${menuItem.name.replace(' ', '')}`;
-            if (!menuItem.sortOrder) {
-              menuItem.sortOrder = 0;
-            }
-            return menuItem;
-          })
           // Sorts ascending on sort order first and then ascending on name
           .sort((a, b) => {
-            if (a.sortOrder > b.sortOrder) {
+            const aSortOrder = _.get(a, 'sortOrder', 0);
```

Comment:
nit: I'd rather see a native implementation than use the lodash dependency. 

maybe
```
const aSortOrder = a.sortOrder || 0;
const bSortOrder = b.sortOrder || 0;
```
related: https://github.com/elastic/kibana/issues/9079